locations/spiders/pilot_flyingj.py

```diff
@@ -1,81 +1,56 @@
-                "PageSize": 50,
-                "States": [],
-                "Countries": [],
-                "Concepts": []
-            }
-        return [scrapy.Request("https://pilotflyingj.com/umbraco/surface/storelocations/search",
-                               method='POST',
-                               body=json.dumps(payload),
-                               headers=self.headers,
-                               callback=self.parse)]
-
-    def parse_store(self, response):
-        properties = response.meta["properties"]
-
-        lat = response.xpath('//ul[@id="places-in-map"]/@data-lat').extract()
-        long = response.xpath('//ul[@id="places-in-map"]/@data-long').extract()
-
-        properties.update({
-            'name': response.xpath('//h1[@class="store-name"]/text()').extract_first().strip(),
-            'website': response.url,
-            'lat':  float(lat[0]),
-            'lon':  float(long[0]),
-            'extras': {
-                'number': properties["ref"]
-            }
-        })
-
-        yield GeojsonPointItem(**properties)
+    start_urls = [
+        'https://pilotflyingj.com/umbraco/surface/storelocations/download?Format=csv&PageSize=1000&PageNumber=1'
+    ]
 
     def parse(self, response):
-
-        data = json.loads(response.body_as_unicode())
-        current_page_number = data["PageNumber"]
-        total_pages = data["TotalPages"]
-
-        stores = data.get("Locations", [])
-
-        if current_page_number <= total_pages:
-            for store in stores:
-                properties = {
-                    'addr_full': store["StreetAddress"].strip(),
-                    'city': store['City'].strip(),
-                    'state': store['State'].strip(),
-                    'postcode': store['ZipCode'].strip(),
-                    'phone': store["PhoneNumber"].strip(),
-                    'ref': store['StoreNumber'],
-                }
-                yield scrapy.Request('https://pilotflyingj.com/stores/{}/'.format(store["Id"]),
-                                     callback=self.parse_store,
-                                     meta={'properties': properties})
-
-            payload = {
-                "PageNumber": current_page_number + 1,
-                "PageSize": 50,
-                "States": [],
-                "Countries": [],
-                "Concepts": []
+        for store in csv.DictReader(response.body_as_unicode().splitlines()):
+            yield GeojsonPointItem(
+                lat=store['Latitude'],
+                lon=store['Longitude'],
+                name=store['Name'],
+                addr_full=store['Address'],
+                city=store['City'],
+                state=store['State/Province'],
+                postcode=store['Zip'],
+                country=store['Country'],
+                phone=store['Phone'],
+                ref=store['Store#'],
+                extras={
+                    'amenity:fuel': True,
```

Comment:
Should this be `'amenity': 'fuel'` to match OSM-ey tags?

Reply:
I was wondering about that myself. I went with this format because we only have one entry per larger place here, instead of individual nodes per feature in OSM. So for something like a Costco, we might want both `amenity=fuel` and `amenity=pharmacy` on the same listing.

I think it would be just as reasonable to go with `amenity=fuel;pharmacy` 🤷‍♂️

Reply:
This is OK for now. I'm glad you're adding this, but I think I'd like to use full-fledged categories instead of OSM tags to specify categorization. We can adjust that later after we decide which set of categories we want to use.

Reply:
👍 I'd be happy to switch stuff over to a more robust categorization system once we figure that part out.